front/public/swagger.json

```diff
@@ -163,8 +163,59 @@
             "schema": {
               "type": "string"
             }
+          },
+          {
+            "in": "query",
+            "name": "use_workspace_credentials",
```

Comment:
@spolu do we want this to appear in the public API?

Reply:
No this is def non documented :+1: